Packs/CommonTypes/ReleaseNotes/3_3_2.md

```diff
@@ -0,0 +1,3 @@
+
+#### Indicator Fields
+- **Key Name**
```

Comment:
I think "Key Values" is a better name as in win registry:
name\key - "Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Zoom\MSI"
data\values - everything else

Also see:
![image](https://user-images.githubusercontent.com/24267931/191459802-c4dc4deb-bd81-4d0a-9f49-fcbd0aa0215d.png)

Reply:
No problem, will change it in the design